clouddriver-kubernetes/src/main/groovy/com/netflix/spinnaker/clouddriver/kubernetes/v1/deploy/ops/servergroup/ResizeKubernetesAtomicOperation.groovy

```diff
@@ -110,17 +108,25 @@ class ResizeKubernetesAtomicOperation implements AtomicOperation<Void> {
     task.updateStatus BASE_PHASE, "Completed resize operation."
   }
 
-  void resizeController(KubernetesV1Credentials credentials, String namespace, String serverGroupName, int size) {
-    if (description.kind == KubernetesUtil.CONTROLLERS_STATEFULSET_KIND) {
+  boolean resizeController(KubernetesV1Credentials credentials, String namespace, String serverGroupName, int size) {
```

Comment:
Why does this return a boolean of whether or not this was deleted? That doesn't seem related to this function.

Reply:
Ping

Reply:
I committed my changes, but this file hadn't been updated yet.  Let me find out what happen.

Reply:
now, code is update-to-date.